src/Transformers/AbstractTransformer.php

```diff
@@ -59,12 +49,8 @@ public function transform($model): array
 
     /**
      * Transform the model relationships to JSON Api output.
-     *
-     * @param mixed $model
-     * @param bool  $inDataTag
-     * @return array transformed relationships
      */
-    protected function transformRelationship($model, $inDataTag = true): array
+    protected function transformRelationship(mixed $model, bool $inDataTag = true): array
```

Comment:
```suggestion
    /**
     * @param TModel $model
     */
    protected function transformRelationship(object $model, bool $inDataTag = true): array
```

Reply:
I left this on purpose, because `$model` can be any related model (this function handles a relationship).